OWNERS

```diff
@@ -1,17 +1,25 @@
-reviewers:
-  - bparees
-  - deads2k
-  - sjenning
-  - smarterclayton
-  - soltysh
-  - tbielawa
-approvers:
-  - bparees
-  - deads2k
-  - derekwaynecarr
-  - eparis
-  - mfojtik
-  - pweil-
-  - sjenning
-  - smarterclayton
-  - tbielawa # also for build and automated release tooling changes
+# See the OWNERS docs at https://go.k8s.io/owners
+
+filters:
+  ".*":
+    reviewers:
+      - bparees
+      - deads2k
+      - sjenning
+      - smarterclayton
+      - soltysh
+      - tbielawa
+    approvers:
+      - bparees
+      - deads2k
+      - derekwaynecarr
+      - eparis
+      - mfojtik
+      - pweil-
+      - sjenning
+      - smarterclayton
+      - tbielawa # also for build and automated release tooling changes
+
+  "vendor/*":
+    labels:
+      - "vendor-update"
```

Comment:
This label will be useful even without rebase so you can e.g. filter out conflicting open PRs touching `vendor/`.

I guess for the rebase we may not even need to modify tide search query, just add another label (`do-not-merge/blocked-paths`) here?

Reply:
I am not sure if it would remove the `do-not-merge/blocked-paths` label automatically after the blockade would be over, so the merge query might be easier.

Reply:
`blockade` plugin will do what you need, yes, when it is reconfigured to remove the `blockade` any event to update the PR will cause us to recalculate whether or not it should have the `do-not-merge/blocked-paths` label and remove if appropriate

Reply:
I was looking at that too. The reason I went for this labeling is that it will be useful to label vendor PRs even if we don't block them so you can easily search for them on GH.